tests/TestEndpoints.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2024 Digital Bazaar, Inc.
+ *
+ * SPDX-License-Identifier: LicenseRef-w3c-3-clause-bsd-license-2008 OR LicenseRef-w3c-test-suite-license-2023
+ */
+
+import {
+  createRequestBody,
+  createVerifyRequestBody
+} from './mock.data.js';
+
+export class TestEndpoints {
+  constructor({implementation, tag}) {
+    this.implementation = implementation;
+    this.tag = tag;
+    this.issuer = implementation.issuers?.find(
+      issuer => issuer.tags.has(tag)) || null;
+    this.verifier = implementation.verifiers?.find(
+      verifier => verifier.tags.has(tag)) || null;
+  }
+  async issue(credential) {
+    const {issuer} = this;
+    const issueBody = createRequestBody({issuer, vc: credential});
+    const response = await post(issuer, issueBody);
+    return response?.verifiableCredential || response;
+  }
+  async verify(vc) {
+    const verifyBody = createVerifyRequestBody({vc});
+    const result = post(this.verifier, verifyBody);
+    if(result?.errors?.length) {
+      throw result.errors[0];
+    }
+    return result;
+  }
+}
+
+export async function post(endpoint, object) {
+  return fetch(endpoint.settings.endpoint, {
+    headers: {
+      'Content-type': 'application/json'
```

Comment:
the content-type is usually `application/json+ld`, but you might know something I don't know WRT to content-type rules.

Reply:
Why are you using fetch here? The existing implementations post will cover oauth2 and zcap based authorization. The use of fetch here gets outside of that which means you will have to recreate adding auth to the request unless in this suite all the endpoints are unprotected?

Reply:
We are definitely **not** sending `application/json+ld`. The vc-api doesn't receive/return `application/ld+json` by design.

Reply:
@aljones15 I've reinstated the vc-test-suite-implementations request module for https class